test/framework/src/test/java/org/elasticsearch/node/MockNodeTests.java

```diff
@@ -33,6 +33,15 @@
 import java.util.List;
 
 public class MockNodeTests extends ESTestCase {
+
+
+    @Override
+    protected boolean enableWarningsCheck() {
+        // As Tribe service as been deprecated, we need to skip deprecation logger
```

Comment:
I did that change because in the `@After` it checks that there are no warnings. I will add an assert on the warning that deprecates the tribe service.